README.md

```diff
@@ -144,23 +143,22 @@ The store has 3 methods:
 
 There are three built-in events:
 
-* `@init` will be fired in `createStoreon`. The best moment to set
-  an initial state.
+* `@init` will be fired in `createStoreon`. Bind to this event to set the initial state.
 * `@dispatch` will be fired on every new action (on `store.dispatch()` calls
-  and `@changed` event). It receives an array with the event name
-  and the event’s data. Can be useful for debugging.
+  and `@changed` events). It receives an array with the event name
+  and the event's data. Can be useful for debugging.
```

Comment:
Thanks for the PR.

Can you just return curly typography?

Reply:
Done!